xonsh/completer.py

```diff
@@ -50,15 +132,19 @@ def startswithlow(x, start, startlow=None):
 
 
 def startswithnorm(x, start, startlow=None):
-    """True if x starts with a string s. Ignores its lowercase version, but
-    matches the API of startswithlow().
+    """True if x starts with a string s.
+
+    Ignores its lowercase version, but matches the API of startswithlow().
     """
+    _ = startlow
```

Comment:
Also note, from the pylint homepage:

```
Fully customizable
Modify your pylintrc to customize which errors or conventions are important to you. The big advantage with Pylint is that it is highly configurable, customizable, and you can easily write a small plugin to add a personal feature.

More about message control (FAQ)
```

I get the feeling that a lot of the issues that are getting brought up are related to trying to enforce conventions that I don't agree with being followed everywhere all of the time, and not with explicit errors or PEP8 violations or violations to the style addendums listed in the dev's guide (http://xonsh.org/devguide.html#python-style-guide)

I would probably be best to disable such convention enforcement for the whole project, if possible.